risc0/cargo-risczero/README.md

```diff
@@ -0,0 +1,32 @@
+# Cargo-risczero
+
+Cargo extension to help create, manage, and test [risc0](https://github.com/risc0/risc0) projects.
+
+## Install
+
+```bash
+## Install from crates.io
+cargo install cargo-risczero
+## Installing from local source
+cargo install --path risc0/cargo-risczero
+```
+
+## New
+
+The `new` command will create a new project for a existing template. It defaults to the template at [rust-starter](https://github.com/risc0/risc0/tree/main/)templates/rust-starter but can be used with other templates locally or hosted on github.
+
+### Examples
+
+```bash
+## Create a project from the main template
+cargo risczero new my_project
+
+## Create a project with 'std' support in the guest
+cargo risczero new my_project --std
+
+## Disable git initialization
+cargo risczero new my_project --no-git
+
+## Create from github template
+cargo risczero new my_project --template https://github.com/risc0/risc0-rust-starter
```

Comment:
Should we add an additional note that this specific example would be for using an old version of our starter template, and that the repo given here is now deprecated?

Or do you think adding a "this is deprecated" warning on https://github.com/risc0/risc0-rust-starter would suffice?

Reply:
IMHO: we should add a DEP notice in the README and then archive the repo. That is a very good visual clue to anyone viewing it.

As for this line: I only picked that url as an example, I could do: `https://github.com/risc0/example` or something else as a dummy value there. For it to be a functional URL that would work I would need to add a additional cli flag which might make it confusing.  I don't know the right solution here.

Reply:
If the repo is clearly marked deprecated -- and good suggestion on also archiving it that should be clearer -- I am now more convinced that just using it as an example here should be ok.